src/setupWorker/start/utils/getWorkerInstance.ts

```diff
@@ -1,13 +1,17 @@
 import { until } from '@open-draft/until'
 import { getWorkerByRegistration } from './getWorkerByRegistration'
-import { ServiceWorkerInstanceTuple } from '../../glossary'
+import {
+  ServiceWorkerInstanceTuple,
+  ServiceWorkerMatcher,
+} from '../../glossary'
 import { getAbsoluteWorkerUrl } from '../../../utils/url/getAbsoluteWorkerUrl'
 
 /**
  * Returns an active Service Worker instance.
  * When not found, registers a new Service Worker.
  */
 export const getWorkerInstance = async (
+  serviceWorkerMatcher: ServiceWorkerMatcher,
```

Comment:
I suggest we add this new argument to the end of the arguments list. Would also look more aligned with other similar function, like `getWorkerByRegistration`.